### PR TITLE
fix: Adding missing AWS rights into the IAM policy

### PR DESCRIPTION
**Pull request by valeriodelsarto**

### CODE REVIEW

File: aws/biganimal-csp-setup [hunk 1]
```diff
@@ -239,7 +239,7 @@ create_iam_policy()
...
...gs:TagLogGroup","logs:UntagLogGroup","s3:CreateBucket","s3:DeleteBucket","s3:DeleteBucketPolicy","s3:DescribeJob","s3:Get*","s3:List*","s3:PutBucketAcl","s3:PutBucketOwnershipControls","s3:PutBucketPolicy","s3:PutBucketPublicAccessBlock","s3:PutBucketTagging","s3:PutBucketVersioning","s3:PutEncryptionConfiguration","s3:PutObject","servicequotas:ListServiceQuotas","sts:DecodeAuthorizationMessage"],"Effect":"Allow","Resource":"*","Sid":"BigAnimalBasePolicy"},{"Action":["ssm:AddTagsToResource","ssm:GetParameters","ssm:PutParameter","ssm:DeleteParameters"],"Effect":"Allow","Resource":"arn:aws:ssm:*:*:parameter/BA_*","Sid":"BigAnimalSystemsManagerParameterStorePolicy"},{"Action":["iam:PassRole","logs:FilterLogEvents"],"Effect":"Allow","Resource":["arn:aws:iam::*:role/ba-${external_id}-*","arn:aws:iam::*:role/biganimal-${suffix_role_name}","arn:aws:logs:*:*:log-group:/ecs/ba-bootstrap:log-stream:"],"Sid":"BigAnimalECSTaskPolicy"},{"Action":"secretsmanager:*","Effect":"Allow","Resource":"arn:aws:secretsmanager:*:*:secret:BA*","Sid":"BigAnimalSecretsPolicy"},{"Sid":"BigAnimalS3Policy","Effect":"Allow","Action":["s3:DeleteObject","s3:DeleteObjectVersion"],"Resource":"arn:aws:s3:::pg-*/*"}]}
```
Comment (nstott) [inline aws/biganimal-csp-setup:242]: LONNNNG lines! 
can we see about prettifying that for easier reviews (sometime?)

**Review by nstott — APPROVED** [review]
lgtm

File: aws/biganimal-csp-setup (hunk 1 above)
Comment (valeriodelsarto) [reply aws/biganimal-csp-setup:242]: yes, the PP version is aws/biganimal_AWS_basepolicy.json, this is why I always update both

Comment (valeriodelsarto) [reply aws/biganimal-csp-setup:242]: is it fine like this?

Comment (nstott) [reply aws/biganimal-csp-setup:242]: fine for now, for sure

**Pull request merged**